joomla/joomla-cms
# [com_languages] submenu changes

##### Description

With the change from https://github.com/joomla/joomla-cms/pull/9101 doesn't make much sense for the languages submenu to have two entries for Installed languages.

This PR, merges them in one submenu option and also adds a submenu in the global admin menu (like exists for extensions).
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/12965643/ab15deea-d04f-11e5-8273-73ab7dbd8d55.png)

![image](https://cloud.githubusercontent.com/assets/9630530/12965626/8a2973fe-d04f-11e5-943b-bbfb7e25fbd2.png)
##### How to test
1. Install latest joomla staging
2. Apply this patch
3. Check language submenu is working properly
